packages/schema/src/config/vite.ts

```diff
@@ -18,9 +18,7 @@ export default defineUntypedSchema({
     mode: {
       $resolve: async (val, get) => val ?? (await get('dev') ? 'development' : 'production')
     },
-    logLevel:{
-      $resolve: async (val, get) => val ?? (await get('dev') ? 'warn' : 'info')
-    },
+    logLevel: 'info',
```

Comment:
should we drop this entirely so we respect top-level `logLevel` instead?
```suggestion
```

Reply:
Good point!